docs/analytics_onboarding/overview.md

```diff
@@ -19,13 +19,14 @@ Below you can find links to the resources themselves, as well as relevant suppor
 **Meetings to Expect - Analysts:**
 Look out for these meetings to be added to your calendar.
```

Comment:
I mention below that some of the deeply nested sections might be useful to collapse into a table (e.g. meetings table, slack channels table). If that's the path we end up using, we can remove the individual docs links from the checklist below, and just link to e.g. the meetings table from this sentence

Reply:
@machow since I've collapsed the nested sections below into tables I just pushed a change that adds a new, single docs link to the table in the sentance above